src/io/flutter/module/FlutterModuleBuilder.java

```diff
@@ -269,7 +269,19 @@ public void dispose() {
 
     @Nullable
     public FlutterSdk getFlutterSdk() {
-      return FlutterSdk.forPath(myPeer.getSdkComboPath());
+      final String sdkPath = myPeer.getSdkComboPath();
+
+      //Ensure the local filesystem has caught up to external processes (e.g., git clone).
+      if (!sdkPath.isEmpty()) {
+        try {
+          LocalFileSystem
+            .getInstance().refreshAndFindFileByPath(sdkPath);
```

Comment:
Should this refresh instead be done at the end of the call to `flutter precache`?